glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -102,13 +102,15 @@ open class GleanInternalAPI internal constructor () {
      * @param applicationContext [Context] to access application features, such
      * as shared preferences
      * @param configuration A Glean [Configuration] object with global settings.
+     * @param uploadEnabled A [Boolean] that determines the initial state of the uploader
```

Comment:
nit: since we moved it in the function signature, let's move it before `configuration` here as well